src/Core/Domain/Order/QueryResult/OrderProductForViewing.php

```diff
@@ -123,6 +123,11 @@ class OrderProductForViewing
      */
     private $orderInvoiceNumber;
 
+    /**
+     * @var bool
+     */
+    private $availableOutOfStock;
```

Comment:
Suggestion for future. For bool primitives suggest to use names like isAvailableOutOfStock or isOutOfStock. Then even without annotations it's clear that this variable is a boolean.

Reply:
imo `is` should be added as a prefix in method only, not in proeperty itself. But I agree that this could be better named, like $availableWhenOutOfStock, but i left it like that unchanged for convienience because another object has the same property.

Reply:
From where comes the rule that "is should be added as a prefix in method only, not in proeperty itself"?

Some suggestions: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
I like @tdavidsonas88 suggestion 👍